### PR TITLE
archivemount: add page

### PR DESCRIPTION
**Pull request by dinaisrael**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by dinaisrael** [comment]
@kbdharun @navarroaxel I've updated the PR according to the tldr-bot suggestion

File: pages/linux/archivemount.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# archivemount
+
+> Mounts an archive for access as a file system.
+> More information: <https://manned.org/archivemount>.
+
+- Mount an archive to mountpoint:
+
+`archivemount {{archive}} {{mountpoint}}`
```
Comment (kbdharun) [inline pages/linux/archivemount.md:8]:
```suggestion
`archivemount {{path/to/archive}} {{mount_point}}`
```
I think this would be better.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by dinaisrael** [comment]
Great suggestion @kbdharun, I've committed the change

File: pages/linux/archivemount.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# archivemount
+
+> Mounts an archive for access as a file system.
```
Comment (kbdharun) [inline pages/linux/archivemount.md:3]:
```suggestion
> Mount an archive for access as a file system.
```

**Comment by dinaisrael** [comment]
Thank you for the feedback @kbdharun, I've committed the change

**Review by kbdharun — APPROVED** [review]

File: pages/linux/archivemount.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# archivemount
+
+> Mount an archive for access as a file system.
+> More information: <https://manned.org/archivemount>.
+
+- Mount an archive to mountpoint:
```
Comment (kbdharun) [inline pages/linux/archivemount.md:6]:
```suggestion
- Mount an archive to a mountpoint:
```
I think this would be better

**Comment by dinaisrael** [comment]
Thank you @kbdharun :) I've committed the additional suggestion

File: pages/linux/archivemount.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# archivemount
+
+> Mount an archive for access as a file system.
```
Comment (navarroaxel) [inline pages/linux/archivemount.md:3]:
```suggestion
> Mount an archive for access as a filesystem.
```

File: pages/linux/archivemount.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://manned.org/archivemount>.
+
+- Mount an archive to a specific mountpoint:
+
+`archivemount {{path/to/archive}} {{mount_point}}`
```
Comment (navarroaxel) [inline pages/linux/archivemount.md:8]: I think this is more clear
```suggestion
`archivemount {{path/to/archive}} {{path/to/mount_point}}`
```

**Comment by dinaisrael** [comment]
Thank you for the feedback @navarroaxel , I've committed the changes

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**